parsec/include/parsec/sys/atomic.h

```diff
@@ -38,13 +38,15 @@ BEGIN_C_DECLS
  * If the compiler provides atomic primitives we prefer to use
  * them instead of our own atomic assembly.
  */
+#    if defined(PARSEC_OSX)
+#       include <AvailabilityMacros.h>
+#    endif
 #    if defined(__FUJITSU)
 #      undef PARSEC_ATOMIC_USE_XLC_32_BUILTINS
 #    endif
 #    if defined(PARSEC_ATOMIC_USE_XLC_32_BUILTINS)
 #      include "atomic-xlc.h"
-#    elif defined(PARSEC_OSX)
```

Comment:
unnecessary changes.

Reply:
Not: if we found OSX, but not the right OSX version, we would define no atomics at all (because we would not consider the other #elif cases for the outer conditional when the inner conditional fails).